### PR TITLE
ServiceNow CMDB

### PR DESCRIPTION
**Pull request by darkushin**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/14034

## Description
Implementing the integration for Service Now CMDB.

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/ServiceNow_CMDB/Integrations/ServiceNow_CMDB/test_data/result_constants.py [hunk 1]
```diff
@@ -0,0 +1,344 @@
...
+                'name': 'Affects Business',
+                'id': '123640000000008033'
+            },
+            'Priority': {
+                'color': '#ff0000',
+                'name': 'High',
+                'id': '123640000000006805'
+            },
+            'CreatedBy': {
+                'email_id': 'akrupnik@paloaltonetworks.com',
```
Comment (Itay4) [inline Packs/ServiceNow_CMDB/Integrations/ServiceNow_CMDB/test_data/result_constants.py:53]: make sure we dont have this kind of sensitive data in the mocks

**Review by bakatzir — COMMENTED** [review]
@Arsenikr Do we want this integration as a new pack? or shall it be part of the `ServiceNow` pack?

File: Packs/ServiceNow/Integrations/ServiceNow_CMDB/ServiceNow_CMDB.py [hunk 2]
```diff
@@ -0,0 +1,534 @@
...
+
+    def delete_relation(self, class_name, sys_id, rel_sys_id, params=None):
+        url_suffix = f'{API_VERSION}{class_name}/{sys_id}/relation/{rel_sys_id}'
+        return self.snow_client.http_request(method='DELETE', url_suffix=url_suffix, params=params)
+
+
+''' HELPER FUNCTIONS '''
+
+
+def handle_sysparms(sys_parms: List, args: dict) -> dict:
```
Comment (yuvalbenshalom) [inline Packs/ServiceNow/Integrations/ServiceNow_CMDB/ServiceNow_CMDB.py:97]: lets talk about this one. not sure if it is necessary.

**Review by yuvalbenshalom — CHANGES_REQUESTED** [review]
this is a preliminary review, see my comments.

File: Packs/ServiceNow/Integrations/ServiceNow_CMDB/ServiceNow_CMDB.yml [hunk 3]
```diff
@@ -0,0 +1,451 @@
...
+  feed: false
+  isfetch: false
+  longRunning: false
+  longRunningPort: false
+  runonce: false
+  script: '-'
+  subtype: python3
+  type: python
+defaultmapperin: HelloWorld-mapper
+defaultclassifier: HelloWorld
```
Comment (yuvalbenshalom) [inline Packs/ServiceNow/Integrations/ServiceNow_CMDB/ServiceNow_CMDB.yml:448]: remove

**Review by yuvalbenshalom — CHANGES_REQUESTED** [review]
good, just remove the hello world leftovers.

**Comment by yuvalbenshalom** [comment]
@darkushin please check the test playbook, its still failing.

**Review by yuvalbenshalom — APPROVED** [review]

**Comment by darkushin** [comment]
@kirbles19 @ShirleyDenkberg This is a new integration for a ServiceNow product we didn't support before. Thank you!

**Comment by ShirleyDenkberg** [comment]
@Itay4 @bakatzir  @yuvalbenshalom 
Doc review completed.

**Pull request merged**